packages/solo/package.json

```diff
@@ -30,7 +30,6 @@
     "@agoric/babel-standalone": "^7.14.3",
     "@agoric/captp": "^1.10.6",
     "@agoric/cosmic-swingset": "^0.34.2",
-    "@agoric/dapp-svelte-wallet": "^0.11.5",
```

Comment:
This `package.json` needs to depend on `@agoric/dapp-wallet` instead.

Reply:
fixed